docs/sources/whatsnew/whats-new-in-v11-0.md

```diff
@@ -310,6 +308,18 @@ Debug or audit using the alert rule metadata and view the alert rule annotations
 
 ![Image shows details of an alert rule](/media/docs/alerting/alert-detail-view.png)
 
+### RBAC for alert rule provisioning APIs
+
+<!-- Yuri Tseretyan -->
+
+_Generally available in all editions of Grafana_
+
+The Alerting Provisioning HTTP API has been updated to enforce Role-Based Access Control (RBAC).
+
+- For Grafana OSS, users with the **Editor** role can now use the API.
+- For Grafana Enterprise and Grafana Cloud, users with the role **Rules Writer** and **Set Provisioning status** can access the API and limit access to alert rules that use a particular data source.
+- Other roles related to provisioning, for example **Access to alert rules** provisioning API still work.
+
 ## Data sources
 
 ### Azure Monitor: Current User authentication
```

Comment:
@imatwawana I'd like to remove this announcement while we figure out the commercial status of Current User Authentication.